dotnet/docs/EXPERIMENTS.md

```diff
@@ -57,13 +57,17 @@ You can use the following diagnostic IDs to ignore warnings or errors for a part
 - SKEXP0052: Memory plugins
 - SKEXP0053: Microsoft 365 plugins
 - SKEXP0054: Web plugins
-- SKEXP0055: Text chunkcer plugin
+- SKEXP0055: Text chunker plugin
 
 ## Planners
 
 - SKEXP0060: Handlebars planner
 - SKEXP0061: OpenAI Stepwise planner
 
+## GoogleAI and VertexAI services
```

Comment:
Suggest Changing the name from `Memory connectors` to `Connectors` as Hugging Face is not only memory. 😊

Add only `SKEXP0033: GoogleVertexAI Connector` and use it in the project level for now.

Reply:
Done